app/_locales/hi/messages.json

```diff
@@ -650,7 +660,7 @@
     "message": "अनुबंध इंटरैक्शन"
   },
   "convertTokenToNFTDescription": {
-    "message": "हमने पाया है कि ये एसेट एक एनएफटी है। MetaMask के पास अब एनएफटी के लिए पूर्ण स्थानीय सपोर्ट है। क्या आप इसे अपनी टोकन सूची से हटाना चाहते हैं और इसे एनएफटी के रूप में जोड़ना चाहते हैं?"
+    "message": "हमने पाया है कि यह संपत्ति एक एनएफटी है। मेटामास्क के पास अब एनएफटी के लिए पूर्ण देशी समर्थन है। क्या आप इसे अपनी टोकन सूची से हटाना चाहते हैं और इसे एनएफटी के रूप में जोड़ना चाहते हैं?"
```

Comment:
Shouldn't we preserve `Metamask` in english here?

Reply:
yes...

Reply:
Checking with our Crowdin rep to fix this. cc @rickycodes